src/app/screens/App/shared/components/LoggedOutHeader.js

```diff
@@ -1,37 +1,34 @@
 import React from 'react';
-import { Navbar, Nav, NavItem, Button, FormGroup, FormControl, InputGroup } from 'react-bootstrap';
+import { Navbar, Nav, NavItem, Button, FormGroup, FormControl } from 'react-bootstrap';
 import { LinkContainer } from 'react-router-bootstrap';
 
 const LoggedOutHeader = () => (
     <Navbar>
         <Nav>
-            <NavItem disabled>Personal</NavItem>
-            <NavItem disabled>Open source</NavItem>
-            <NavItem disabled>Business</NavItem>
-            <NavItem disabled>Explore</NavItem>
-            <NavItem disabled>Help</NavItem>
+            <NavItem>Personal</NavItem>
+            <NavItem>Open Source</NavItem>
+            <NavItem>Business</NavItem>
+            <NavItem>Explore</NavItem>
+            <NavItem>Personal</NavItem>
```

Comment:
I started fresh and just didn't really realize they were disabled but anyway their URL is pointing to '#'  which effectively disables it.